srcpkgs/libsecret/template

```diff
@@ -1,24 +1,20 @@
 # Template file for 'libsecret'
 pkgname=libsecret
-version=0.20.4
+version=0.20.5
 revision=1
-build_style=gnu-configure
+build_style=meson
 build_helper="gir"
-configure_args="--enable-introspection --enable-vala --disable-static
- --with-libgcrypt-prefix=${XBPS_CROSS_BASE}/usr"
-hostmakedepends="automake glib-devel gtk-doc pkg-config libxslt vala"
+hostmakedepends="glib-devel pkg-config libxslt vala gi-docgen docbook-xsl
+ gettext"
 makedepends="glib-devel libgcrypt-devel vala"
 short_desc="GObject based library for accessing the Secret Service API"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="http://www.gnome.org"
```

Comment:
I think we can figure out a better homepage, maybe `https://wiki.gnome.org/Projects/Libsecret`?

Reply:
that's good